.werft/observability/monitoring-satellite.ts

```diff
@@ -82,6 +82,16 @@ export class MonitoringSatelliteInstaller {
                     requests: { memory: '200Mi', cpu: '50m' },
                 },
             },
+            remoteWrite: {
+                username: '${process.env.PROM_REMOTE_WRITE_USER}',
+                password: '${process.env.PROM_REMOTE_WRITE_PASSWORD}',
+                urls: ['https://prometheus.gitpod-dev.com/api/v1/write'],
+                writeRelabelConfigs: [{
+                    sourceLabels: ['__name__'],
+                    regex: 'probe_.*|rest_client_requests_total',
```

Comment:
I've just noticed that this regex drops the `up` metric, which is required [by SLI 2](https://github.com/gitpod-io/ops/issues/2765). 

The `up` metric is built from ALL targets, so I need to figure out a good regex that will only give us the up from probers...